docs/Installation/Installation_Guide.rst

```diff
@@ -102,298 +79,6 @@ If you are behind a proxy, add the following lines to your local machine’s
     https_proxy=http://user;pass@<my-ip>:<my-port>

```

Comment:
This proxy stuff can be deleted as well. No clue where it came from and nothing at the application level depends on it AFAIK.